providers/implementations/signature/mac_legacy.c

```diff
@@ -172,9 +172,13 @@ static void *mac_dupctx(void *vpmacctx)
         return NULL;
 
     *dstctx = *srcctx;
+	dstctx->propq = NULL;
```

Comment:
Something strange going on with the indentation here

Reply:
Replaced the tab with spaces in a new commit